script/release-ts.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
this script does the following:
1) gets current latest release
2) runs semantic release which creates a new release for ts and marks it as latest (github default behaviour)
3) marks back prev latest release as current latest

Reply:
lets get this script under top level `script` dir

Reply:
it is there already

Reply:
blind